### DISCUSSION TITLE
SaluSpa Palm Springs AirJet  60234 (Costco model)

### DISCUSSION THREAD

**Discussion by n3xus1** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#746] [General]
The SaluSpa Palm Springs AirJet  60234 (Costco model) works with the D1 mini 
Pump S100104
Firmware: 2024-06-17-1334
Model: MIAMI2021
CIO: 6 wire, 2021 (air)
DSP: 6 wire, 2021 (air)

I used custom pinouts as I'm using the D1 mini.

Couple of questions about calibration, how do you do the manual and virtual calibrations?

Also is there a login option with user name and password, if so how do you access it and set it up.

I'm thinking of adding a ambient temperature sensor, would port D1 or D8 work? anyone do any write ups or have more informations.

Thanks to everyone for their hardwork and efforts

I also added it in home assistant using a card template I found on the discussions 

![IMG_7296](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/6356263/ff6ec8f4-9543-47cd-b4a3-dc5c990e1ae5)

**Comment by visualapproach** [comment]
The calibration should be described in the manual and in the tooltips on the web page. Authentication to access the web ui is not used for the moment. I discovered today that it was removed at some stage. But it should be trivial to implement I hope. Check future updates..
I think D0 can be used but you have a custom build so you have to test yourself.

**Reply by n3xus1** [reply]
it may be worth adding this model to the support pdf as it was not listed, I'd assume if it works with a D1 mini it will work with original design.
I decided to skip the ambient sensor and just pull the local temperature from home assistant via mqtt.

thanks again for all your effort into this project

**Comment by jasii** [comment]
@n3xus1 Could I please get a bit more information on your custom pinout and what you did to install this? I have the same model as you and I have a d1 mini free that I'd like to try this with and would like to try and build it to save some money. From your photo it seems like you don't need the level shifter and the resistors with the d1 mini and it's working fine.

**Comment by n3xus1** [comment]
sorry for the late reply, I kept the wires short about 2-3" long. No level shifter or resistors and it works great.
I'm running the latest dev software on it.
see setting below

![Screenshot 2024-08-19 at 7 10 13 PM](https://github.com/user-attachments/assets/0e387d0c-d8fd-4cf4-bb3b-0840bc1756f6)
